contracts/SafientMain.sol

```diff
@@ -217,6 +245,36 @@ contract SafientMain is IArbitrable, IEvidence {
         emit CreateSafe(msg.sender, _inheritor, metaEvidenceID);
     }
 
+    function syncSafe(
+        address _creator,
+        string memory _safeId,
+        string calldata _metaEvidence
+    ) external payable syncSafeRequisite(_creator, _safeId, _metaEvidence) {
+        metaEvidenceID += 1;
+
```

Comment:
The safe creation proof part is yet to be worked on, right?

Reply:
Yeah, I wanted to discuss that once before going forward. I was planning to do it in the next release.